packages/core/src/audio/audio.controller.ts

```diff
@@ -0,0 +1,27 @@
+import { ReactiveControllerHost } from 'lit';
+import { Audible } from './audio.element.js';
+
+/**
+ *
+ * What would be interesting here is adding a robust config option like we have with clarity
+ * motion. Clarity motion works off of transitions, which is fine, but audio would need to
```

Comment:
One idea I had on Friday when I was hacking up this branch was that the audio context is singletonish and the sounds are functions that can be called as needed. The config objects could be used to call the sound function.